### PR TITLE
Update Debian builds from 11.6 to 11.7

### PR DESCRIPTION
**Pull request by mmguero**
New ISOs and sha256 sums are at https://cdimage.debian.org/debian-cd/current/amd64/iso-dvd/ and https://cdimage.debian.org/debian-cd/current/arm64/iso-dvd/

## Description
Updates Debian from 11.6 to 11.7 release, released April 29, 2023.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: os_pkrvars/debian/debian-11-aarch64.pkrvars.hcl
Comment (mmguero) [inline os_pkrvars/debian/debian-11-aarch64.pkrvars.hcl:1]: see https://cdimage.debian.org/debian-cd/current/arm64/iso-dvd/SHA256SUMS

File: os_pkrvars/debian/debian-11-x86_64.pkrvars.hcl
Comment (mmguero) [inline os_pkrvars/debian/debian-11-x86_64.pkrvars.hcl:1]: see https://cdimage.debian.org/debian-cd/current/amd64/iso-dvd/SHA256SUMS

**Review by mmguero — COMMENTED** [review]
Linked to sha256sum files for review

**Review by Stromweld — APPROVED** [review]
Looks good to me

**Pull request merged**